drivers/hwinfo/Kconfig

```diff
@@ -21,6 +21,30 @@ config HWINFO_SHELL
 	help
 	  Enable hwinfo Shell for testing.
 
+config HWINFO_CC13XX_CC26XX
+	bool "TI SimpleLink CC13xx/CC26xx hwinfo driver"
+	default y
+	depends on SOC_SERIES_CC13X2_CC26X2 || SOC_SERIES_CC13X2X7_CC26X2X7
+	help
+	  Enable TI SimpleLink CC13xx/CC26xx hwinfo driver.
+
+config HWINFO_CC13XX_CC26XX_USE_FACTORY
+	bool "Ignore CCFG and always use factory value"
+	default y
+	depends on HWINFO_CC13XX_CC26XX
+	help
+	  Always return factory pre-programmed values, stored in FCFG and ignore
+	  values provided in CCFG (Customer Configuration).
+
+config HWINFO_CC13XX_CC26XX_USE_BLE_MAC
```

Comment:
You may (optionally) consider using a choice value for this with two options: BLE vs. 15.4. IMO that would be easier to understand for the end user. To me it's not immediately obvious why BLE and/or 15.4 MAC should be preferred.

Reply:
I switched to `menuconfig` for the whole driver Kconfig symbol and `choice` for BLE/IEEE MAC selection, thanks!